artifacts.go

```diff
@@ -0,0 +1,33 @@
+package jenkins_client
```

Comment:
I prefer to use `pkg` as the name of the package. `jenkins_client` is too long.

Reply:
Considering Jenkins client will be used by others, I moved `client/*` to /*

Reply:
How about move those files into the directory `pkg`?

Reply:
That's great.